base_tier_validation/models/tier_validation.py

```diff
@@ -161,6 +187,11 @@ def _get_rejected_message(self):
         )
         return self.validation_status == "rejected" and msg or ""
 
+    def _compute_validated_rejected(self):
+        for rec in self:
+            rec.validated = self._calc_reviews_validated(rec.review_ids)
+            rec.rejected = self._calc_reviews_rejected(rec.review_ids)
```

Comment:
Here too, the value should be derived from validation_status. I'm surprised to see the `_calc_reviews` methods still exist as they are not used to compute the validation_status.

Reply:
According to what is mentioned at https://github.com/OCA/server-ux/pull/1084#discussion_r2243345108, the `_calc_reviews` methods were maintained. Should they also be removed? Personally, I don't understand the need to “remove methods” in some cases (perhaps breaking compatibility if someone is using them), but on the other hand, it is necessary to keep these `validated`/`rejected` fields and/or not make them store (the migration scripts have already defined the appropriate values).

Reply:
Or, alternatively you could have kept `_calc_reviews` to *determine* validation status, but it was decided to replace this functionality by the new methods `_validated_states` and `_rejected_states`. For full backwards compatibility, you could choose to revert this. It depends on whether the `_calc_reviews` methods are overridden regularly in depending modules I guess.

Reply:
(from what I can see, the only override of a `_calc_reviews` method was in base_tier_validation_forward and it was correctly refactored out in https://github.com/OCA/server-ux/pull/1084 already, so I think it's safe to remove the `_calc_reviews` methods entirely.

Reply:
If this change and https://github.com/OCA/server-ux/pull/1137#discussion_r2254045634 are correct, I will apply them to PR 17.0.